tests/integration/servicecatalog/cmd_service_test.go

```diff
@@ -37,6 +37,13 @@ var _ = Describe("odo service command tests", func() {
 		})
 	})
 
+	Context("checking machine readable output for service catalog", func() {
+		It("should succeed listing catalog components", func() {
+			// Since service catalog is constantly changing, we simply check to see if this command passes.. rather than checking the JSON each time.
+			helper.CmdShouldPass("odo", "catalog", "list", "services", "-o", "json")
```

Comment:
Instead of matching the whole json can you match the basic key value pair like ```"kind": "ServiceList"``` and atleast one ```"items": [``` you mentioned in pr description.